assets/scripts/util/fetch_nonblocking.js

```diff
@@ -0,0 +1,206 @@
+/* global abortEverything, saveStreetIncomplete */
+import $ from 'jquery'
+
+export let serverContacted
```

Comment:
Actually, this one is harder because `serverContacted` is set in the global bundle.

I'm going to propose moving it to `app/initialization.js` for now. One of the things we should think through is a system for storing application state.